addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1646,7 +1646,8 @@ const Wysiwyg = Widget.extend({
                             this.odooEditor.document.getSelection().collapseToStart();
                         }
                         this._updateEditorUI(this.lastMediaClicked && { target: this.lastMediaClicked });
-                        colorpicker.off('color_leave');
+                        colorpicker.off('color_leave', null, ev => {
+                        });
```

Comment:
I don't think this is correct. The selection is collapsed at the start, but why do we need to collapse it if we're just calling deselectTable?

I would just change 
```js
if (this.odooEditor.deselectTable() && hasValidSelection(this.odooEditor.editable)) {
    this.odooEditor.document.getSelection().collapseToStart();
}
```
to
```js
if (hasValidSelection(this.odooEditor.editable)) {
    this.odooEditor.deselectTable();
}
```